File: VendorResponseSystem/ViewModels/UserModel.cs
namespace VendorResponseSystem.ViewModels
{
    public class UserModel
    {
        public string UserName { get; set; }
        public string DisplayName { get; set; }
        public string Description { get; set; }
        public string TeamName { get; set; }
        public string Email { get; set; }
        public bool IsMemberOfVendorResponse { get; set; }
        public bool IsMemberOfVendorResponseAdmin { get; set; }
        public string BuyerCodes { get; set; }
    }
}

File: VendorResponseSystem/ClientApp/src/models/user-model.ts
export class UserModel {
    public constructor(userName: string, displayName: string, description: string, teamName: string, email: string){
        this.userName = userName;
        this.displayName = displayName;
        this.description = description;
        this.teamName = teamName;
        this.email = email;
    }
    userName! : string;
    displayName! : string;
    description! : string;
    teamName! : string;
    email! : string;
}
